app/views/profiles/_group_membership.html.erb

```diff
@@ -4,7 +4,7 @@
     <% own_memberships do |membership| %>
       <li class="uk-padding-small uk-clearfix">
           <div class="uk-padding-remove uk-float-left uk-width-medium-1-3">
-            <%= link_to membership.group_name, group_path(membership.group_id) %>
+            <%= membership.group_link %>
```

Comment:
It could be `membership.membership.group.decorate.link`.
So something wrong here.